<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regpage</class>
 <widget class="QDialog" name="regpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iCardSIS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/src.qrc">
    <normaloff>:/images/2.jpg</normaloff>:/images/2.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="Email">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>380</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {background-color: grey;
color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;
border:3px grey;
border-radius:10px;}
QLineEdit:focus{background-color: black;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>480</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {background-color: grey;
color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;
border:3px grey;
border-radius:10px;}
QLineEdit:focus{background-color: black;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Enter Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>610</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 131, 131);

font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneNumber">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {background-color: grey;
color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;
border:3px grey;
border-radius:10px;}
QLineEdit:focus{background-color: black;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>311</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images/src.qrc">:/images/2.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: grey;
color: rgb(255, 255, 255);
font: 700 18pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string> Registration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="conPass">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>530</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {background-color: grey;
color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;
border:3px grey;
border-radius:10px;}
QLineEdit:focus{background-color: black;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserName">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {background-color: grey;
color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;
border:3px grey;
border-radius:10px;}
QLineEdit:focus{background-color: black;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images/src.qrc">:/images/1.jpg</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images/src.qrc"/>
 </resources>
 <connections/>
</ui>
